app/src/components/QuestMap.vue

```diff
@@ -45,30 +75,31 @@ async function getProgress() {
 }
 
 const genOption = (nodes, edges) => ({
-  rendener: 'svg',
+  render: 'svg',
```

Comment:
I think it should be `renderer`?

Reply:
Sorry that I deleted it because ChatGPT recommand me to do it